readme.md

```diff
@@ -119,14 +135,25 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+First one will not throw an error and thus fail silently. The second will obviously do the opposite and give you some information back as to the problem or acknowledge that the save was not sucessful.
 
 ## Question 10
 
 Using jQuery, write an AJAX request to `http://tunr.com/artists` that would create a new artist with the name of 'Resin Laying Deer Figurine, Gold', and pop up a box saying "All done!" when complete.
 
 ```js
-// Your code starts here...
+var url = `http://tunr.com/artists`;
+function addArtist(){
+  $.ajax({
+    url: url,
+    type: 'POST',
+    data: { json: JSON.stringify({name:'Resin Laying Deer Figurine, Gold'})},
```

Comment:
dont believe you have to be extra explicit about the format of the data, i.e.
```
  data: {
    name: "Resin Laying Deer Figurine, Gold"
  }
```
should work as well, but like the your consideration